app/src/main/java/com/framgia/fsalon/screen/booking/BookingPresenter.java

```diff
@@ -8,13 +8,16 @@
 import com.framgia.fsalon.data.model.DateBooking;
 import com.framgia.fsalon.data.model.Salon;
 import com.framgia.fsalon.data.model.Stylist;
+import com.framgia.fsalon.data.model.UserRespone;
 import com.framgia.fsalon.data.source.BookingRepository;
 import com.framgia.fsalon.data.source.SalonRepository;
 import com.framgia.fsalon.data.source.StylistRepository;
+import com.framgia.fsalon.data.source.UserRepository;
 
 import java.util.ArrayList;
 import java.util.List;
 
+import io.reactivex.Observable;
```

Comment:
ERROR: Unused import: io.reactivex.Observable. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports